src/tests/multihost/alltests/test_services.py

```diff
@@ -88,15 +88,17 @@ def test_0003_bz1713368(self, multihost):
         """
         # sssd-dbus is a weak dependency of sssd-tools package so we did not
         # get sssd-dbus in '# yum deplist sssd-tools' command
-        rpm_grep = 'yum repoquery --recommends sssd-tools'
-        cmd = multihost.client[0].run_command(rpm_grep, raiseonerr=False)
+        version = float(re.findall("\d+\.\d+", multihost.client[0].distro)[0])
+        if version >= 9:
+            rpm_grep = 'yum repoquery --requires sssd-tools'
+            cmd = multihost.client[0].run_command(rpm_grep, raiseonerr=False)
+        else:
+            rpm_grep = 'yum repoquery --recommends sssd-tools'
+            cmd = multihost.client[0].run_command(rpm_grep, raiseonerr=False)
```

Comment:
line 94 and 97 are repeats, it can be out of the conditional

Reply:
Fixed